packages/create-cloudflare/src/templates.ts

```diff
@@ -410,11 +415,10 @@ const downloadRemoteTemplate = async (src: string) => {
 
 		const tmpDir = await mkdtemp(join(tmpdir(), "c3-template"));
 		await emitter.clone(tmpDir);
-		s.stop(`${brandColor("template")} ${dim("cloned and validated")}`);
```

Comment:
do we need to update the status here as well?

Reply:
oh, I think you're doing that [here](https://github.com/cloudflare/workers-sdk/pull/6306/files#diff-0717b6ed1b4a85c41d325f24981721e346254c785696159aec9c0e444d768efeR330) instead

Reply:
Yep. I moved it out because it wasn't validated yet.